File: NotificationSystem/NotificationSystem.Data/Classes/clsAgent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using NotificationSystem.NotificationSystem.Data.NotificationSystemTableAdapters;


namespace NotificationSystem.NotificationSystem.Data.Classes
{


    public class clsAgent
    {
		internal int Original_AgentID;
		private string m_Title = string.Empty;
        private string m_FirstN = string.Empty;
        private string m_LastN = string.Empty;
        private string m_Email = string.Empty;
        private string m_Phone = string.Empty;
        private string m_Fax = string.Empty;

        private int m_AgentID = 0;
        private int m_TroubleTicketNo = 0;
        private DateTime m_StartDate;
        private int m_Salary;
        public string Title
        {
            get
            {
                return m_Title;
            }
            set
            {
                m_Title = value;
            }
        }


        public string FirstN
        {
            get
            {
                return m_FirstN;
            }
            set
            {
                m_FirstN = value;
            }
        }

        public string LastN
        {
            get
            {
                return m_LastN;
            }
            set
            {
                m_LastN = value;
            }
        }

        public string Email
        {
            get
            {
                return m_Email;
            }
            set
            {
                m_Email = value;
            }
        }

        public string Phone
        {
            get
            {
                return m_Phone;
            }
            set
            {
                m_Phone = value;
            }
        }

        public string Fax
        {
            get
            {
                return m_Fax;
            }
            set
            {
                m_Fax = value;
            }
        }

        public int AgentID
        {
            get
            {
                return m_AgentID;
            }
            set
            {
                m_AgentID = value;
            }
        }


        public int TroubleTicketNo
        {
            get
            {
                return m_TroubleTicketNo;
            }
            set
            {
                m_TroubleTicketNo = value;
            }
        }

        public DateTime StartDate
        {
            get
            {
                return m_StartDate;
            }
            set
            {
                m_StartDate = value;
            }
        }

        public int Salary
        {
            get
            {
                return m_Salary;
            }
            set
            {
                m_Salary = value;
            }
        }

    }
}

File: NotificationSystem/NotificationSystem.Data/Classes/clsAgent.vb


Namespace NotificationSystem.NotificationSystem.Data.Classes


    Public Class clsAgent
        Friend Property Original_AgentID As Integer = 0
        Private m_Title As String = String.Empty
        Private m_FirstN As String = String.Empty
        Private m_LastN As String = String.Empty
        Private m_Email As String = String.Empty
        Private m_Phone As String = String.Empty
        Private m_Fax As String = String.Empty
        Private m_Salary As String = String.Empty

        Private m_AgentID As Integer = 0
        Private m_TroubleTicketNo As Integer = 0
        Private m_StartDate As Date

        Public Property Title As String
            Get
                Return m_Title
            End Get
            Set(value As String)
                m_Title = value
            End Set
        End Property


        Public Property FirstN As String
            Get
                Return m_FirstN
            End Get
            Set(value As String)
                m_FirstN = value
            End Set
        End Property

        Public Property LastN As String
            Get
                Return m_LastN
            End Get
            Set(value As String)
                m_LastN = value
            End Set
        End Property

        Public Property Email As String
            Get
                Return m_Email
            End Get
            Set(value As String)
                m_Email = value
            End Set
        End Property

        Public Property Phone As String
            Get
                Return m_Phone
            End Get
            Set(value As String)
                m_Phone = value
            End Set
        End Property

        Public Property Fax As String
            Get
                Return m_Fax
            End Get
            Set(value As String)
                m_Fax = value
            End Set
        End Property

        Public Property AgentID As Integer
            Get
                Return m_AgentID
            End Get
            Set(value As Integer)
                m_AgentID = value
            End Set
        End Property


        Public Property TroubleTicketNo As Integer
            Get
                Return m_TroubleTicketNo
            End Get
            Set(value As Integer)
                m_TroubleTicketNo = value
            End Set
        End Property

        Public Property StartDate As Date
            Get
                Return m_StartDate
            End Get
            Set(value As Date)
                m_StartDate = value
            End Set
        End Property

        Public Property Salary As String
            Get
                Return m_Salary
            End Get
            Set(value As String)
                m_Salary = value
            End Set
        End Property

    End Class
End Namespace
